powerscale/provider/namespace_acl_datasource_test.go

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright (c) 2024 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Mozilla Public License Version 2 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://mozilla.org/MPL/2/
+
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provider
+
+import (
+	"fmt"
+	"regexp"
+	"terraform-provider-powerscale/powerscale/helper"
+	"testing"
+
+	"github.com/bytedance/mockey"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccNamespaceAclDataSource(t *testing.T) {
+	var namespaceACLTerraformName = "data.powerscale_namespace_acl.test"
```

Comment:
@RayLiu7 The acceptance test case naming convention demands NamespaceACL instead of NamespaceAcl as part of test methods. If we want to leverage the acceptance test tag to just run NamespaceACL test cases.
@P-Cao correct me if my understanding is wrong.

Reply:
It's okay to use NamspaceAcl. It only has the prefix restriction "TestAcc". In this case, we can try to use "run e2e test NamespaceAclDataSource"